src/analysis/souffle/tests/arcs_manifest_tests_todo/BUILD

```diff
@@ -14,8 +14,6 @@
 # limitations under the License.
 #
 #-------------------------------------------------------------------------------
-load("//build_defs:arcs.bzl", "arcs_manifest_proto")
```

Comment:
This whole `arcs_manifest_tests_todo` directory can be deleted. It contains arcs manifests that we eventually wanted to turn into tests. But since the arcs frontend is no longer being supported, there's no reason to keep them around.